frontend/workflows/projectSelector/src/project-group.tsx

```diff
@@ -219,6 +226,15 @@ const ProjectGroup: React.FC<ProjectGroupProps> = ({ title, group, displayToggle
                   )}
                 </StyledClearIcon>
               </StyledHoverOptions>
+              <StyledFlexEnd hidden={key !== keyWithQLinksOpen}>
+                {state?.projectData?.key?.linkGroups && (
```

Comment:
Should this be wrapped around the `StyledFlexEnd`? That way we dont even render the entire element if there aren't any links